### ISSUE TITLE
Add differential translation analysis

### ISSUE THREAD

**Issue by pinin4fjords** [nf-core/riboseq#27]
### Description of feature

RiboDiff? RiboTaper? Babel?

**Comment by JackCurragh** [comment]
@pinin4fjords Are we happy with Anota2Seq as the solution to this? The outstanding limitation as far as I can see it is the requirement for 3 replicates which is not quite something we could aim to solve.

I believe [DeltaTE](https://doi.org/10.1002/cpmb.108) may be more flexible

**Comment by pinin4fjords** [comment]
Yes, think it's probably covered as I envisaged, someone else can open feature requests for evolutions
